lib/smart_app_launch/token_introspection_access_token_group.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'standalone_launch_group_stu2'
+
+module SMARTAppLaunch
+  class TokenIntrospectionAccessTokenGroup < Inferno::TestGroup
+    title 'Request New Access Token to Introspect'
+    run_as_group
+
+    id :token_introspection_access_token_group
```

Comment:
I'd put `smart` at the start of all the ids.

Reply:
renamed